android-client-sdk/src/main/java/com/devcycle/sdk/android/eventsource/EventHandler.kt

```diff
@@ -0,0 +1,71 @@
+package com.devcycle.sdk.android.eventsource
+
+import timber.log.Timber
+import com.devcycle.sdk.android.eventsource.EventHandler
+
+class EventHandle: EventHandler {
```

Comment:
should we change this to `EventHandler`

Reply:
It doesn't seem to like having a class with the same name as the interface. Perhaps naming it something like `DVCEventHandler` would make sense? Not sure what the convention is here

Reply:
oh I see. Nvm then. Adam called the iOS equivalent just `Handler` so we could go with that